### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by jbampton**
### Description

This PR fixes spelling

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

File: api/src/main/java/org/apache/cloudstack/api/command/admin/network/CreateStorageNetworkIpRangeCmd.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public class CreateStorageNetworkIpRangeCmd extends BaseAsyncCmd {
 
     @Parameter(name = ApiConstants.VLAN,
                type = CommandType.INTEGER,
-               description = "Optional. The vlan the ip range sits on, default to Null when it is not specified which means you network is not on any Vlan. This is mainly for Vmware as other hypervisors can directly reterive bridge from pyhsical network traffic type table")
+               description = "Optional. The vlan the ip range sits on, default to Null when it is not specified which means you network is not on any Vlan. This is mainly for Vmware as other hypervisors can directly retrieve bridge from physical network traffic type table")
```
Comment (stephankruggg) [inline api/src/main/java/org/apache/cloudstack/api/command/admin/network/CreateStorageNetworkIpRangeCmd.java:67]:
```suggestion
               description = "Optional. The vlan the ip range sits on, default to Null when it is not specified which means your network is not on any Vlan. This is mainly for Vmware as other hypervisors can directly retrieve bridge from physical network traffic type table")
```

File: api/src/main/java/org/apache/cloudstack/api/response/LBStickinessPolicyResponse.java [hunk 2]
```diff
@@ -55,8 +55,8 @@ public class LBStickinessPolicyResponse extends BaseResponse {
     @Param(description = "is policy for display to the regular user", since = "4.4", authorized = {RoleType.Admin})
     private Boolean forDisplay;
 
-    // FIXME : if prams with the same name exists more then once then value are concatinated with ":" as delimiter .
-    // Reason: Map does not support duplicate keys, need to look for the alernate data structure
+    // FIXME : if prams with the same name exists more then once then value are concatenated with ":" as delimiter .
```
Comment (stephankruggg) [inline api/src/main/java/org/apache/cloudstack/api/response/LBStickinessPolicyResponse.java:58]:
```suggestion
    // FIXME : if params with the same name exist more than once then values are concatenated with ":" as delimiter .
```

**Review by stephankruggg — APPROVED** [review]

**Review by BryanMLima — APPROVED** [review]

**Review by GaOrtiga — APPROVED** [review]

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by RodrigoDLopez — APPROVED** [review]

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
This PR has a merge problem in the package jobs. I am try it manually but so far have not reproduced the problem.

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@jbampton I've reverted the spelling change in git-pr so packages can build

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by JoaoJandre — APPROVED** [review]
LGTM

**Pull request merged**